packages/ciphernode/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM node:20 AS evm-builder
```

Comment:
LTS is 22 now I believe - should we keep this current or is there a reason to stick with an older version